### DISCUSSION TITLE
How to add a new item to the existing menu?

### DISCUSSION THREAD

**Discussion by hoanhtuan-hat** [fedorovvl/tso_client#49] [Q&A] [answered]
I have several questions that need help after downloading the VS Studio and downloading this project successfully (I have experience in C# Desktop before):
- How to add a new menu item, since I can not find the Menu anywhere?
- How to decode the message send in and out. I have an idea that "Auto build Mining", check the mine if exists and automatically build the mine, and want to try.

Thank you

**Comment by fedorovvl** [comment, marked as answer]
check https://raw.githubusercontent.com/fedorovvl/tso_client/master/client/user.js
first line is what you need =)
menu logic is not c#.. its adobe air nativemenu used in javascript
